content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -6,8 +6,15 @@ weight: 70
 
 <!-- overview -->
 
-The Konnectivity service provides a TCP level proxy for the control plane to cluster
-communication.
+Kubernetes can use an optional proxy mechanism, _Konnectivity_, that helps you connect 
+your control plane's {{< glossary_tooltip text="API servers" term_id="kube-apiserver" >}} 
+to your nodes, to {{< glossary_tooltip term_id="etcd" >}}, or to aggregated API servers.  
```

Comment:
can't the konnectivity service be used for all three egress types: controlplane (authn, authz, admission, conversion webhooks), etcd, and cluster (self-hosted aggregated apiservers)?